client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -53,28 +53,6 @@ Undocumented.prototype.getInboundTransferStatus = function ( domain, fn ) {
 	);
 };
 
-/**
- * Starts an inbound domain transfer that is in the pending_start state.
- *
- * @param {number|string} siteId The site ID
- * @param {string} domain The domain name
- * @param {string} authCode The auth code for the transfer
- * @param {Function} fn The callback function
- * @returns {Promise} A promise that resolves when the request completes
- */
-Undocumented.prototype.startInboundTransfer = function ( siteId, domain, authCode, fn ) {
-	let query = {};
-	if ( authCode && authCode !== '' ) {
```

Comment:
`authCode !== ''` was practically unnecessary here, because `authCode` already ensures that it's not an empty string. Glad we've cleaned that up 👍